templates/repo/commits_list_small.tmpl

```diff
@@ -30,23 +32,14 @@
 					{{$class = (printf "%s%s" $class " isWarning")}}
 				{{end}}
 			{{end}}
-			{{if $.comment.Issue.PullRequest.BaseRepo.Name}}
-				<a href="{{$.comment.Issue.PullRequest.BaseRepo.Link}}/commit/{{PathEscape .ID.String}}" rel="nofollow" class="{{$class}}">
-			{{else}}
-				<span class="{{$class}}">
```

Comment:
Under what condition was that block false?
I don't find any that makes sense, so I'd say this is a reasonable simplification…